setup.cfg

```diff
@@ -36,6 +36,8 @@ classifiers =
   Programming Language :: Python :: 3.8
   Programming Language :: Python :: 3.9
   Programming Language :: Python :: 3.10
+  Programming Language :: Python :: 3.11
+  Programming Language :: Python :: 3.12
```

Comment:
It was previously decided not to put these in, as the CI is still not passing on Windows/Mac, so 3.11+ support is still considered experimental.